src/v2/Apps/Settings/Routes/DeleteAccount/DeleteAccountRoute.tsx

```diff
@@ -29,18 +39,25 @@ export const DeleteAccountRoute: FC = () => {
         <Formik
           validateOnMount
           initialValues={{
-            explanation: "",
             confirmation: false,
+            explanation: "",
+            hasPassword: hasPassword,
```

Comment:
not sure if this is good practice. it's not meant to be a field in the form. i added it so that `password` field's validation can be disabled if `hasPassword` is false.

Reply:
This is fine. Formik is for forms, but more abstractly, it's a slick transitional state management lib. 

Note that in JS there's no need to double up props like this if the key is the same name as the variable. See [object short-hand](https://attacomsian.com/blog/javascript-object-property-shorthand).

Reply:
good to know. thanks @damassi. revised.